bdk-ffi/src/electrum.rs

```diff
@@ -16,11 +16,16 @@ use bdk_wallet::KeychainKind;
 use std::collections::BTreeMap;
 use std::sync::Arc;
 
-pub struct ElectrumClient(BdkBlockingClient);
+// NOTE: We are keeping our naming convention where the alias of the inner type is the Rust type
```

Comment:
Good note, too bad, but totally get it and its proper for our codebase's conventions for sure